sycl/test/basic_tests/context_platforms.cpp

```diff
@@ -0,0 +1,41 @@
+// RUN: %clangxx -fsycl %s -o %t.out
+// RUN: %t.out
+#include <CL/sycl.hpp>
+#include <iostream>
+
+using namespace cl::sycl;
+
+int main(int argc, char const *argv[]) {
+  vector_class<platform> platforms = platform::get_platforms();
+  if (platforms.size() < 2) {
+    std::cout << " This test requires at least 2 SYCL platforms to be present "
+                 "on the system. Skipping it"
+              << std::endl;
+    return 0;
+  }
+  auto AsyncHandler = [](exception_list ES) {
+    for (auto &E : ES) {
+      std::rethrow_exception(E);
+    }
+  };
+  queue Q(AsyncHandler);
+  try {
+    vector_class<device> all_platforms_devices;
+    for (unsigned i = 0; i < platforms.size(); i++) {
+      vector_class<device> one_platform_devices = platforms[i].get_devices();
+      all_platforms_devices.push_back(one_platform_devices[0]);
+    }
+    context all_context = context(all_platforms_devices);
+
+    Q.wait_and_throw();
```

Comment:
You haven't submitted any commands to the queue, so, no need to wait for anything
```suggestion
```

Reply:
Done